admin/templates/configuration/tmp/configure/sql/QueryMetadata.sql

```diff
@@ -95,6 +95,20 @@ CREATE TABLE IF NOT EXISTS `QWorker` (
 ENGINE = InnoDB
 COMMENT = 'Mapping of queries to workers';
 
+-- -----------------------------------------------------
+-- Table `QStatsTmp`
+-- MEMORY table - will be recreated(but empty) by mariadb every time server starts. 
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `QStatsTmp` (
+  `queryId` BIGINT NOT NULL COMMENT 'Query ID',
+  `totalChunks` INT COMMENT 'Total number of cunks in the query',
+  `completedChunks` INT COMMENT 'Number of completed chunks in the query',
+  `queryBegin` TIMESTAMP DEFAULT 0 COMMENT 'When the query was started', 
```

Comment:
Does `queryBegin` keep the same (or about the same) timestamp as `submitted` in `QInfo` table? Maybe get rid of it if it's not used?